docs/src/pages/components/no-ssr/no-ssr.md

```diff
@@ -21,7 +21,7 @@ This component can be useful in a variety of situations:
 
 ## Frame deferring
 
-In it's core, the NoSsr component purpose is to **defer rendering**.
+In its core, the NoSsr component purpose is to **defer rendering**.
```

Comment:
This sentence seems a bit weird to me but it’s missing an apostrophe on component I.e component’s

Reply:
```suggestion
At its core, the NoSsr component's purpose is to **defer rendering**.
```
(As an aside, I've also never been a big fan of the excessive and unnecessary bold text.)